syntax = "proto3";

package pb;

message UserRequest {
    int64 id = 3;
    string username = 2;
    string image = 1;
}

message UserResponse {}

message IDRequest {
    int64 id = 1;
}

service Users {
    rpc AddUser (UserRequest) returns (UserResponse);
    rpc ChangeUser (UserRequest) returns (UserResponse);
    rpc DeleteUser(IDRequest) returns (UserResponse);
}